website/docs/tutorials/scaffoldTutorial.mdx

```diff
@@ -0,0 +1,220 @@
+  },
+  "eslintConfig": {
+    "extends": [
+      "react-app",
+      "react-app/jest"
+    ]
+  },
+  "browserslist": {
+    "production": [
+      ...
+    ],
+    "development": [
+      ...
+    ]
+  },
+  "proxy": "http://localhost:20000"
+}
+```
+
+As you can see, each project has a different set of dependencies, scripts, and settings. The `package.json` files are used to specify the dependencies, scripts, and settings for each project. This means each project can be built, started, and used independently
+
+#### Root Project Settings
+
+The `package.json` file in the project root is used to bundle both apps together into a single node app that can be run from a single command. It contains only the scripts needed to run both apps and looks like this:
+```json
+{
+  "private": true,
+  "scripts": {
+    "setup": "npm run setup:taqueria && npm run setup:app",
+    "setup:taqueria": "cd ./taqueria && npm install && taq init",
+    "setup:app": "cd ./app && npm install",
+    "start": "npm run build:taqueria && npm run start:app",
+    "build:taqueria": "cd ./taqueria && npm run build",
+    "start:app": "cd ./app && npm run start",
+    "start:local": "npm run start:taqueria:local && npm run start:app",
+    "start:taqueria:local": "cd ./taqueria && npm run start:local",
+    "stop:local": "npm run stop:taqueria:local",
+    "stop:taqueria:local": "cd ./taqueria && npm run stop:local"
+  }
+}
+```
+
+## Building and Starting the Project
+
+The root Node project contains `setup` and `start` scripts which can be run together on a single line to build, deploy, and launch the dapp. This process does the following steps: 
+1. Install npm dependencies
+2. Run the required commands to build and deploy the Taqueria project
+3. Start up the React app
+
+From the root of the project, run the following command to build and start the project:
+```bash
+npm run setup && npm run start
+```
+This will result in your browser opening to a simple webpage that has a `connect` button for the Beacon wallet integration
+
+It's important here to understand what steps occur implicitly during this so you can understand how the different plugins and files are in Taqueria
+
+### Building and Deploying the Taqueria Project
+
+If you inspect the scripts in `/taqueria/package.json`, you will see the following actions are executed when the setup command is run:
+1. Build the `/taqueria` NPM project: `npm run build` - installs npm dependencies, including Taqueria plugins
+2. Compile the ligo contract: `taq compile` - uses the LIGO Taqueria plugin to compile the smart contract located in `/taqueria/contracts`, outputing compiled Michelson `.tz` files in the `/artifacts` directory
+3. Generates TS types: `taq generate types` - uses the Taqueria Contract Types plugin to generate TS types from the compiled smart contract, and install them into the React app (`/app/types`) for use with Taquito in the frontend
+
+#### Compiling the Smart Contract
+
```

Comment:
Are these other sections going to be added in the future?

Reply:
Yes, this doc is blocked until the state API is implemented as in order to use the generated types with real contract addresses, we need to be able to store the address and pass it into the front end app 

This file actually shouldn't be in this PR as it's covered in #640